mappings/net/minecraft/util/ReusableStream.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS zx net/minecraft/util/ReusableStream
+	CLASS zx$1
+		FIELD b pos I
+		METHOD tryAdvance (Ljava/util/function/Consumer;)Z
+			ARG 1 consumer
+	FIELD a collectedElements Ljava/util/List;
+	FIELD b source Ljava/util/Spliterator;
+	METHOD <init> (Ljava/util/stream/Stream;)V
+		ARG 1 stream
+	METHOD a getStream ()Ljava/util/stream/Stream;
```

Comment:
`newStream` as mentioned in the issue.

Reply:
Why not just .stream()?

Reply:
I was just going by what Marcono1234 said in the original comment. I was expecting a static method here, but looking at the code `stream` is the better name.

Reply:
Done